apps/dune-mock/src/endpoints/dune-mock/dune-mock.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { Body, Controller, Get, HttpException, Param, Post } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { CreateTableBody } from "./entities";
+import { DuneMockService } from "@libs/services/dune-mock";
+import { CsvFile } from "./entities/csv.file";
+
+@Controller('/dune-mock')
+@ApiTags('dune-mock')
+export class DuneMockController {
+    constructor(
+        private readonly duneMockService: DuneMockService,
+    ) { }
+
+    @Post("/table/create")
+    async createTable(
+        @Body() body: CreateTableBody,
+    ): Promise<HttpException> {
+        const isTableCreated = await this.duneMockService.createTable(body);
+        if (isTableCreated) {
+            return new HttpException("Table was created sucessfully !", 201);
+        } else {
+            throw new HttpException("Table was not created !", 400);
+        }
+    }
+
+    @Post("/:table_name/insert")
+    async insertIntoTable(
+        @Param('table_name') tableName: string,
+        @Body() body: CsvFile,
+    ): Promise<HttpException> {
+        const isDataInserted = await this.duneMockService.insertIntoTable(tableName, body);
+        if (isDataInserted) {
+            return new HttpException("Data was inserted succesfully !", 201);
+        } else {
+            throw new HttpException("Table not found !", 404);
+        }
+    }
+
+    @Get("/generate-charts")
```

Comment:
make it POST instead of GET (harder to call, but correct way)

Reply:
it was used just for a test, i deleted it